config/src/config/network_config.rs

```diff
@@ -331,6 +331,8 @@ pub struct RateLimitConfig {
     pub ip_byte_bucket_size: usize,
     /// Initial amount of tokens initially in the bucket
     pub initial_bucket_fill_percentage: u8,
+    /// Allow for an open throttle
+    pub open: bool,
```

Comment:
maybe rename this to `enabled`? seems more intuitive to me when writing the config, like

```yaml
inbound_rate_limit_config:
    enabled: false
```